dodola/cli.py

```diff
@@ -51,6 +51,27 @@ def dodola_cli(debug):
     logging.basicConfig(level=loglevel)
 
 
+@dodola_cli.command(help="Clean up and standardize GCM")
+@click.argument("x", required=True)
+@click.argument("out", required=True)
+@click.option(
+    "--drop_leapdays/--no-drop_leapdays",
+    default=True,
+    help="Whether to remove leap days",
+)
+def cleancmip6(x, out, drop_leapdays):
+    """Clean and standardize CMIP6 GCM to 'out'. If drop-leapdays option is set, remove leap days"""
+    services.clean_cmip6(x, out, drop_leapdays)
+
+
+@dodola_cli.command(help="Remove leap days and update calendar")
+@click.argument("x", required=True)
+@click.argument("out", required=True)
+def removeleapdays(x, out):
+    """ Remove leap days and update calendar attribute"""
+    services.remove_leapdays(x, out)
```

Comment:
Like above, need to authenticate storage and pass it to the service here. Nothing is getting passed to the required `storage` arg.